components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1289,8 +1289,9 @@ void RewardsServiceImpl::LoadURL(
   if (test_response_callback_) {
     std::string test_response;
     std::map<std::string, std::string> test_headers;
-    test_response_callback_.Run(url, &test_response, &test_headers),
-    callback(200, test_response, test_headers);
+    int code = 200;
```

Comment:
We should use response status codes from `src/net/http/http_status_code.h`, i.e. `HTTP_OK`

Reply:
done